resources/testing/veteran_reviews_count.csv

```diff
@@ -0,0 +1,7 @@
+name,vca_link,No. of Reviews
+Andreas,https://docs.google.com,10

```

Comment:
I know it's very simple but I would at least try with different n. of reviews 😂

Reply:
Well, i did it on porpouse, I wanted it to be squared 😅 , but yeah, let me fix some values.